### DISCUSSION TITLE
How to merge cell of header?

### DISCUSSION THREAD

**Discussion by quoc27690** [nick-keller/react-datasheet-grid#112] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58845804/150623425-f690d477-7c37-4458-b8ef-2ecd2364b47a.png)
I want to merge cell of header. Can it work as image ?

**Comment by nick-keller** [comment, marked as answer]
Hello @quoc27690 ,
Short answer: no. Multiline headers is not supported, nor is merging.
I'd assume you could add a class to your headers using `headerClassName` on the column and tweak the CSS to make it span multiple columns, but that would mess with all the click behaviors.

**Reply by alpharaoh** [reply]
Is there any chance of adding support for both multiline headers and merging? It is important for my use case.

**Reply by nick-keller** [reply]
Not planned. This adds a lot of complexity that is not balanced with enough added value so far.
